catalog/models.py

```diff
@@ -142,25 +142,25 @@ def tagged(self, tags):
         return self.get_queryset().tagged(tags)
 
 
-# @todo #257:30m  Document terms Product, Position and Option
-#  Seems the best way is to do it with subclassing.
-#  Such documentation will clarify types and will be obvious for programmers.
-#  But check the docs too. May be some of this terms are mentioned there.
-class AbstractProduct(models.Model, AdminTreeDisplayMixin):
+# @todo #261:60m  Use the new catalog arch at sites.
+class Position(models.Model):
     """
-    Product model.
-    Defines basic functionality and primitives for Product in typical e-shop.
-    Has n:1 relation with Category.
+    The smallest unique catalog item.
+
+    By catalog semantics position has is the description of phisical product variation.
```

Comment:
```suggestion
    In catalog semantics, position is a description of physical variation of a product.
```

Reply:
> of phisical product variation.

i'll leave this and will fix all another